app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,22 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_accessor :decoded_token, :jwt_token
+      attr_reader :authenticator_name, :service_id, :account, :username, :client_ip, :request
+
+      def initialize(authentication_input:, jwt_token:)
+        @authenticator_name = authentication_input.authenticator_name
+        @service_id = authentication_input.service_id
+        @account = authentication_input.account
+        @username = authentication_input.username
+        @client_ip = authentication_input.client_ip
+        @jwt_token = jwt_token
+      end
+
+      def authn_jwt_variable_id
+        "#{@account}:variable:conjur/#{@authenticator_name}/#{@service_id}"
```

Comment:
what is this variable used for? i didn't find it initialized in a policy in the cucumber test.